pkg/apis/batch/types.go

```diff
@@ -128,6 +128,14 @@ type JobSpec struct {
 	// +optional
 	Completions *int32
 
+	// Specifies the policy of handling failed pods. In particular, it allows to
```

Comment:
You need to add defaulting in `pkg/apis/batch/v1` and validation in `pkg/apis/batch/validation`

Reply:
Validation added, I don't see any need for defaulting